### DISCUSSION TITLE
p\project\trendup\app\Multipliers" directory does not exist.

### DISCUSSION THREAD

**Discussion by Temian1** [cjmellor/level-up#9] [Q&A]
i tried to give user point and i got that issue

```
p\project\trendup\app\Multipliers" directory does not exist.
```

**Comment by cjmellor** [comment]
Possible bug I've overlooked.

For now you manually add `app\Multipliers` folder

**Comment by Temian1** [comment]
Second issue I encountered was , point can be added until you have created levels, I think you should make level optional

**Comment by cjmellor** [comment]
Unlikely to add that. Levelling up is a big part of the package

**Comment by Temian1** [comment]
I would suggest you do Like this

If level was not set or created

When a point is to be added , automatically create a default level

**Comment by cjmellor** [comment]
Yeah, good idea to be fair. Gonna do a PR for that.

**Reply by cjmellor** [reply]
Covered in #14
